pyvista/utilities/fileio.py

```diff
@@ -233,3 +235,58 @@ def read_exodus(filename,
 
     reader.Update()
     return pyvista.wrap(reader.GetOutput())
+
+
+def read_meshio(filename, **kwargs):
+    """Read any mesh file using meshio."""
+    # Import meshio
+    import meshio
+    from meshio._vtk import (
+        meshio_to_vtk_type,
+        vtk_type_to_numnodes,
+    )
+    
+    # Read mesh file
+    mesh = meshio.read(filename, **kwargs)
+
+    # Extract cells from meshio.Mesh object
+    offset = []
+    cells = []
+    cell_type = []
+    cell_data = {}
+    for k, v in mesh.cells.items():
+        vtk_type = meshio_to_vtk_type[k]
+        numnodes = vtk_type_to_numnodes[vtk_type]
+        offset += [len(offset)+i*(numnodes+1) for i in range(len(v))]
+        cells += numpy.hstack((numpy.full((len(v), 1), numnodes), v)).ravel().tolist()
```

Comment:
The `.tolist()` doesn't seem to be necessary. Just `numpy.concatenate` instead of `array` after the loop.